docs/cli/product-exporter.md

```diff
@@ -1,6 +1,6 @@
 # Product Exporter
 
-A package that helps with exporting [commercetools products](https://dev.commercetools.com/http-api-projects-products.html) from the [commercetools platform](http://dev.commercetools.com/).
+A package that helps with exporting [commercetools products](https://docs.commercetools.com/http-api-projects-products.html#product) from the [commercetools platform](http://dev.commercetools.com/).
```

Comment:
👍 
On a side note: are all other `.md` updated to references the new URL?
We should change those, although not here

Reply:
It's already merged to master